### ISSUE TITLE
warnings during build

### ISSUE THREAD

**Issue by cdietrich** [eclipse/xtext-umbrella#69]
```
[INFO] Building zip: /var/jenkins_home/workspace/xtext-umbrella_master-TJVK2XMFM2B7NDWEDHK3TC4G5FUVJYPBOHKPDANCAGTIOETSMFMA/releng/org.eclipse.xtext.sdk.p2-repository/target/org.eclipse.xtext.sdk.p2-repository-2.15.0-SNAPSHOT.zip
[INFO] 
[INFO] --- tycho-p2-repository-plugin:1.2.0:assemble-repository (default) @ org.eclipse.xtext.sdk.p2-repository ---

[WARNING] Mirror tool: Problems resolving provisioning plan.: [Unable to satisfy dependency from v20180625-1629.Default 1.0.0.888C77cMGiYEEB7sQfpRFhfMp8hL to org.eclipse.equinox.p2.iu; org.junit [4.11.0.v201303080030,4.11.0.v201303080030].]
[WARNING] More information on the preceding warning(s) can be found here:
[WARNING] - http://wiki.eclipse.org/Tycho_Messages_Explained#Mirror_tool
[WARNING] Mirror tool: Problems resolving provisioning plan.: [Unable to satisfy dependency from v20180625-1629.Default 1.0.0.888C77cMGiYEEB7sQfpRFhfMp8hL to org.eclipse.equinox.p2.iu; org.junit [4.11.0.v201303080030,4.11.0.v201303080030].]
[WARNING] More information on the preceding warning(s) can be found here:
[WARNING] - http://wiki.eclipse.org/Tycho_Messages_Explained#Mirror_tool
[WARNING] Mirror tool: Problems resolving provisioning plan.: [Unable to satisfy dependency from v20180625-1629.Default 1.0.0.888C77cMGiYEEB7sQfpRFhfMp8hL to org.eclipse.equinox.p2.iu; org.junit [4.11.0.v201303080030,4.11.0.v201303080030].]
[WARNING] More information on the preceding warning(s) can be found here:
[WARNING] - http://wiki.eclipse.org/Tycho_Messages_Explained#Mirror_tool

[INFO] 
```

**Comment by cdietrich** [comment]
i wonder where this dependency is coming from

**Comment by cdietrich** [comment]
Fixed in 2.15
